runtime/src/main/java/org/corfudb/util/Utils.java

```diff
@@ -16,6 +16,7 @@
 import java.text.DecimalFormat;
 import java.util.Collections;
 import java.util.Comparator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Comparator'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)